app/kumactl/data/install/k8s/control-plane/crds/kuma.io_circuitbreaker.yaml

```diff
@@ -10,385 +10,386 @@ spec:
   names:
     kind: CircuitBreaker
     plural: circuitbreakers
-  scope: ""
-  validation:
-    openAPIV3Schema:
-      description: CircuitBreaker is the Schema for the circuitbreaker API
-      properties:
-        apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
-          type: string
-        kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
-          type: string
-        metadata:
+  scope: Namespaced
```

Comment:
Please keep these as `scope: Cluster`. All our resources are not namespace bound.

Reply:
Hey, thanks for the review @nickolaev -- these are set to empty strings in the current version, which I thought defaults to "Namespaced". Am I missing something?

Reply:
The only one that is explicitly set to `Cluster` is `Mesh`.

Refs:
https://github.com/Kong/kuma/blob/f46bc59eeb58f9f7d5295a7c2135aee31d7def0e/app/kumactl/data/install/k8s/control-plane/crds/kuma.io_circuitbreaker.yaml#L10-L13

https://github.com/Kong/kuma/blob/f46bc59eeb58f9f7d5295a7c2135aee31d7def0e/app/kumactl/data/install/k8s/control-plane/crds/kuma.io_meshes.yaml#L10-L13

Reply:
Yes, that is true. On the other hand, we only access these from `kuma-system` (i.e. where the kuma-cp is deployed) so maybe `Namespaced` is good enough. Let's go with it.